### PR TITLE
Feature/brpa 714 add events trigger

### PR DESCRIPTION
**Pull request by felipefadul**
#### What is the purpose of this pull request?

- Pixel events trigger added: AddToCart, ViewContent and PlaceAnOrder.

#### How to test it?

You will need the TikTok Pixel Helper extension for Chrome and a valid pixel ID set in the app's settings. You can access [this workspace](https://tiktoktbp--sandboxbrdev.myvtex.com/) and see the events being triggered by accessing pages, accessing product pages (product params are passed) and adding to cart (items params are also passed).

#### Screenshots or example usage

image

image

image

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/resolvers/tradePolicies.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export async function tradePolicies(
 
   const log = loggerService(logger)
   const service = salesChannelService(salesChannel, log)
-  const tradePolice = await service.getTradePolicies()
+  const tradePolicie = await service.getTradePolicies()
```
Comment (ConradoClark) [inline node/resolvers/tradePolicies.ts:20]:
```suggestion
  const tradePolicies = await service.getTradePolicies()
```

File: node/resolvers/tradePolicies.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export async function tradePolicies(
...
 
-  return tradePolice
+  return tradePolicie
```
Comment (ConradoClark) [inline node/resolvers/tradePolicies.ts:22]:
```suggestion
  return tradePolicies
```

File: node/services/salesChannelService.ts [hunk 3]
```diff
@@ -13,10 +13,10 @@ export const salesChannelService = (
...
       const response = await salesChannelData.getSalesChannelList()
 
-      const tradePolicies = response.map((polices: GetSalesChannel) => ({
-        id: polices.Id,
-        name: polices.Name,
-        currency: polices.CurrencyCode,
+      const tradePolicies = response.map((policies: GetSalesChannel) => ({
+        id: policies.Id,
+        name: policies.Name,
+        currency: policies.CurrencyCode,
```
Comment (ConradoClark) [inline node/services/salesChannelService.ts:19]:
```suggestion
      const tradePolicies = response.map((policy: GetSalesChannel) => ({
        id: policy.Id,
        name: policy.Name,
        currency: policy.CurrencyCode,
```

**Comment by ConradoClark** [comment]
The comments above are just a few suggestions for the naming adjustments 😄

**Comment by felipefadul** [comment]
Thanks, Conrado!

**Review by ConradoClark — APPROVED** [review]
LGTM!

**Review by MIzidio — APPROVED** [review]

**Pull request merged**